openslides/motions/static/templates/motions/motion-list.html

```diff
@@ -233,7 +233,9 @@ <h1 translate>Motions</h1>
                     ng-options="tag.id as tag.name for tag in tags"
                     search-contains="true"
                     id="selectTag"
-                    class="form-control">
+                    class="form-control"
+                    data-placeholder-text-single="'Select some tags' | translate"
```

Comment:
I think we should better use `placeholder-text-multiple` because we use a multiple select field here, see https://github.com/leocaseiro/angular-chosen/issues/121 and http://plnkr.co/edit/P3bWW2?p=preview
Can you try it this works?